src/dlgConnectionProfiles.cpp

```diff
@@ -1706,13 +1706,15 @@ void dlgConnectionProfiles::slot_connectToServer()
     }
 
     if (needsGenericPackagesInstall) {
-        //install generic mapper script
+        //install appropriate mapper script for the game
         if (pHost->getUrl().contains(QStringLiteral("aetolia.com"), Qt::CaseInsensitive) || pHost->getUrl().contains(QStringLiteral("achaea.com"), Qt::CaseInsensitive)
             || pHost->getUrl().contains(QStringLiteral("lusternia.com"), Qt::CaseInsensitive)
             || pHost->getUrl().contains(QStringLiteral("imperian.com"), Qt::CaseInsensitive)) {
```

Comment:
Does **IRE**'s ***Starmourn*** need to have this mapper? Or the new generic one?

Reply:
Starmourn uses the IRE mapper.

Reply:
So is this a convenient point to do that also?

*\*Ponders changing PR title to:* "Pre-install correct mapper"